sources/mysql/infoschema.go

```diff
@@ -288,10 +293,10 @@ func (isi InfoSchemaImpl) GetForeignKeys(conv *internal.Conv, table common.Schem
 	for _, k := range keyNames {
 		foreignKeys = append(foreignKeys,
 			schema.ForeignKey{
-				Name:         fKeys[k].Name,
-				Columns:      fKeys[k].Cols,
-				ReferTable:   fKeys[k].Table,
-				ReferColumns: fKeys[k].Refcols})
+				Name:           fKeys[k].Name,
+				ColIds:         fKeys[k].Cols,
+				ReferTableId:   fKeys[k].Table,
+				ReferColumnIds: fKeys[k].Refcols})
 	}
```

Comment:
We are currently working on assigning Id at core layer, instead of having a common function to assign id. So this should be resolved when those changes are added.